manual/manual/library/libunix.etex

```diff
@@ -47,9 +49,10 @@ more information on the functions that are not supported under Windows.
 This module is identical to "Unix"~(\ref{Unix}), and only differs by
 the addition of labels. You may see these labels directly by looking
 at "unixLabels.mli", or by using the "ocamlbrowser" tool.
-\fi
```

Comment:
By curiosity, why did you move the end of the if after \newpage?

Reply:
The "newpage" was clearly there to be used only in the case of the printed document slurping in the Unix API docs inline. I knew it had no effect in the HTML documents but it seemed to me to be cleaner to "document" that it was intended for the printed documentation by moving it.

Reply:
Good point.